1.14.5/bullseye/Dockerfile

```diff
@@ -33,11 +33,13 @@ RUN apt update && apt install --no-install-recommends -y \
     gpg-agent \
     && rm -rf /var/lib/apt/lists/*
 
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```

Comment:
Agree with removing `set -ex`, for now I'm not adding `-u` because I'm not convinced that "no variable checks" is better than what we have right now, we can look at that separately though. Will adjust.